tests/legacy/unit/commands/test_extensions.py

```diff
@@ -65,6 +65,62 @@ def test_list_extensions(self, fake_iter_modules):
             ),
         )
 
+    @mock.patch(
+        "pkgutil.iter_modules",
+        return_value=(
+            (None, "test1", None),
+            (None, "test2", None),
+            (None, "test3", None),
+            (None, "_test4", None),
+        ),
+    )
+    def test_list_extensions_verbose(self, fake_iter_modules):
```

Comment:
The purpose of the test changes is to ensure we added support to all commands. If we didn't, these tests would fail.

I would prefer to parameterize rather than duplicating existing tests, but these are `unitTest` fixtures and not worth the effort/risk of refactoring.